cli/node/mod.rs

```diff
@@ -285,17 +285,41 @@ pub fn node_resolve_npm_reference(
   Ok(Some(resolve_response))
 }
 
+pub fn node_resolve_binary_commands(
+  pkg_nv: &NpmPackageNv,
+  npm_resolver: &NpmPackageResolver,
+) -> Result<Vec<String>, AnyError> {
+  let package_folder =
+    npm_resolver.resolve_package_folder_from_deno_module(pkg_nv)?;
+  let package_json_path = package_folder.join("package.json");
+  let package_json = PackageJson::load(
+    npm_resolver,
+    &mut PermissionsContainer::allow_all(),
+    package_json_path,
+  )?;
+
+  Ok(match package_json.bin {
+    Some(Value::String(_)) => vec![pkg_nv.name.to_string()],
```

Comment:
Didn't know that `scripts` supported a single string 🤔

Reply:
This is the package.json `bin` entry.